Apps/FCM/.bundle/config

```diff
@@ -0,0 +1,2 @@
+BUNDLE_PATH: "vendor/bundle"
```

Comment:
what is this file and should it be removed or added to gitignore?

Reply:
I'm not sure. This was added by [React Native initialization template](https://github.com/customerio/customerio-reactnative/pull/182/commits/5a5c39a62027b37e7ebda566c2d1285c686252ef) so I haven't changed it. But I don't think this needs to be in `gitignore` because the [`gitignore` file generated by React Native template](https://github.com/customerio/customerio-reactnative/pull/182/commits/5a5c39a62027b37e7ebda566c2d1285c686252ef#diff-435c869fe6b5c3f8b071c2f3f3facbd61f8a47907fec7827e0d3c9dd5ed9b124) doesn't include it.